conf/routes

```diff
@@ -22,14 +22,6 @@ DELETE   /dslinks               controllers.MainController.removeDisconnectedLin
 PUT      /uplinks               facades.websocket.WebSocketController.uplinkHandshake(url, name)
 DELETE   /uplinks/:name         facades.websocket.WebSocketController.removeUplink(name)
 
-# Metrics
-GET      /metrics/cluster       controllers.MetricController.memberEvents(role: Option[String], address: Option[String], from: Option[Long], to: Option[Long])
-GET      /metrics/connections   controllers.MetricController.connectionEvents(linkName: Option[String], from: Option[Long], to: Option[Long])
-GET      /metrics/sessions      controllers.MetricController.sessionEvents(linkName: Option[String], from: Option[Long], to: Option[Long])
-GET      /metrics/requests      controllers.MetricController.requestStatsByLink(from: Option[Long], to: Option[Long])
-GET      /metrics/batches       controllers.MetricController.requestStatsByMethod(from: Option[Long], to: Option[Long])
-GET      /metrics/responses     controllers.MetricController.responseStatsByLink(from: Option[Long], to: Option[Long])
```

Comment:
Are we confirmed from our PO delete "old" rest api for metrics?
Am i got right that we provide new Rest API for metric purpose? How could i see those API?

Reply:
That functionality used only for metrics/statistics collection. I discussed it with @uralian, we desided that it's not a problem to return this functionality if we'll need it, but now it's looks unnecessary.

Reply:
And yes, old api removed since we send metrics right to the graphite server